modules/flowable-engine-common/src/main/resources/org/flowable/common/db/mapping/changeTenantId.xml

```diff
@@ -0,0 +1,54 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<!--
+    ~ Licensed under the Apache License, Version 2.0 (the "License");
+    ~ you may not use this file except in compliance with the License.
+    ~ You may obtain a copy of the License at
+    ~
+    ~       http://www.apache.org/licenses/LICENSE-2.0
+    ~
+    ~ Unless required by applicable law or agreed to in writing, software
+    ~ distributed under the License is distributed on an "AS IS" BASIS,
+    ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    ~ See the License for the specific language governing permissions and
+    ~ limitations under the License.
+-->
+
+<!DOCTYPE mapper PUBLIC "-//mybatis.org//DTD Mapper 3.0//EN" "http://mybatis.org/dtd/mybatis-3-mapper.dtd">
+
+<mapper namespace="org.flowable.common.db.mapping">
+
+    <sql id="changeTenantIdSetSql">
+        <set>
+            TENANT_ID_ = #{targetTenantId, jdbcType=VARCHAR}
+        </set>
+    </sql>
+    
+    <sql id="changeTenantIdCriteriaSql">
+        where RES.TENANT_ID_ = #{sourceTenantId, jdbcType=VARCHAR}
```

Comment:
This will not work on Oracle if the `sourceTenantId` is the default empty tenant. In Oracle an empty tenant is stored as null. Therefore the where clause will always be `false`. This is also applicable for the other places where you use the `TENANT_ID_` in the SQL.